tests/sles4sap/hana_install.pm

```diff
@@ -89,7 +83,7 @@ sub run {
     }
     assert_script_run "df -h";
 
-    # Check we have an hdblcm
+    # Check if we have a hdblcm
```

Comment:
IMO, in this case the correct article should be `an` due to the way the h in `hdblcm` is pronounced as if starting with a vowel. Wouldn't normally comment on a comment, but the previous one seems to be better grammatically. I think we can reword further the comment to avoid an a-vs-an situation as: `hdblcm is used for installation. Verify it exists`

Reply:
OK, I will change the comment.